doc/user/op_guide.md

```diff
@@ -1,5 +1,7 @@
 # Operation Guide
 
+etcd-operator is available as a [Helm chart](https://github.com/kubernetes/charts/tree/master/stable/etcd-operator).
```

Comment:
can we move this into the installation section as an alternative section?

or we can create a helm section at the bottom of the doc.